indico/modules/events/registration/templates/display/regform_list.html

```diff
@@ -10,6 +10,15 @@
     {% trans %}Available forms{% endtrans %}
 {% endblock %}
 
+{% block description %}
+    {% set description = event_settings.get(event, 'registration_description') %}
+    {% if description %}
+        <div class="registration-description">
+            {{ description | safe }}
```

Comment:
This introduces an XSS vulnerability...

Reply:
For HTML the correct thing to use here would be the `sanitize_html` filter, and for markdown it's the `markdown` filter.